### DISCUSSION TITLE
Why Laragon latest version doesn't have latest Postgress and MongoDB versions ?

### DISCUSSION THREAD

**Discussion by Raefat** [leokhoa/laragon#562] [General]
Why Laragon latest version doesn't have latest Postgress and MongoDB versions, and is there is a way to get a simple packages.ini that has all links for latest versions please let me know

**Comment by RobiNN1** [comment]
In Laragon, you can update everything yourself. It doesn't need to be updated constantly like xampp..

**Reply by Raefat** [reply]
yea i know, i just want to see if there's a config file that has all links to the latest versions, I don't wanna search every time for Postgres and MySQL and MongoDB links and update them and so on.

**Reply by Pen-y-Fan** [reply]
When I started to learn PHP I thought I had to be on the latest version, over time I've realised I only need to be on the same minor version (PHP 8.1, 8.2 etc). The latest release of each is less critical. The same goes for Apache, MySQL etc. I'm fine as long as I have the same minor version as production.

There is value in automating the installation of packages, not included with Laragon. For example, there is a way to add PostgreSQL using **packages.conf** (Laragon menu > Tools > Quick Add > configuration) see #386 

Theoretically, if the Laragon community was interested, a [Wiki](https://github.com/leokhoa/laragon/wiki) page could be created with the latest links. Although, in my opinion, the links don't _need_ to be updated that regularly. Anyone who identifies a new release _could_ edit the page.

Is this what you were thinking?
